### PR TITLE
Enable R 4.0 builds

### PR DESCRIPTION
**Pull request by okankcb**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by okankcb** [comment]
@conda-forge-admin please rerender

**Comment by okankcb** [comment]
@conda-forge/r

File: .azure-pipelines/azure-pipelines-win.yml
Comment (mfansler) [inline .azure-pipelines/azure-pipelines-win.yml:1]: You should not manually edit any rendered files. The feedstock should be in fully working condition after a rerender.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+r_base:
+ - 4.2
+ - 4.1
+ - 4.0
```
Comment (mfansler) [inline recipe/conda_build_config.yaml]: Conda Forge does not support R 4.2 for **win** platform. Please follow two space indentation for YAML files.

You must rerender after merging this suggestion:

```suggestion
  - 4.2  # [not win]
  - 4.1
  - 4.0
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Please use informative commit messages and describe the intent of this PR with a proper title and a brief statement.

**Comment by okankcb** [comment]
@conda-forge-admin please rerender

**Comment by okankcb** [comment]
@conda-forge-admin please rerender

**Review by mfansler — APPROVED** [review]

**Pull request merged**